src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -53,6 +53,11 @@ typedef void (^MTRDeviceConnectionCallback)(MTRBaseDevice * _Nullable device, NS
 @property (readonly, nonatomic, nullable)
     NSNumber * controllerNodeID API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
 
+/**
+ * If true, the controller is browsing for commissionables.
+ */
+@property (readonly, nonatomic, getter=isBrowsing) BOOL browsing MTR_NEWLY_AVAILABLE;
```

Comment:
I think we should name this browsingForComissionables, in case we ever want to expose operational browse on here.

That said, if we are only exposing this for test purposes, do we want to do that?

Reply:
Removed.